app/views/content/landing/how-to-fund-your-teacher-training/_collage.html.erb

```diff
@@ -3,7 +3,7 @@
     <section class="col col-full-content">
       <div class="inset">
         <p class="col-720">
-          If you train to be a teacher, you could be eligible to receive a scholarship or bursary to help you train.
+          If you train to be a teacher, you could be eligible to receive a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> to help you train.
```

Comment:
```suggestion
          If you train to be a teacher, you could be eligible to get a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> to help you train.
```

I know this was the prod wording previously - worth simplifying a bit?

Reply:
```suggestion
          If you train to be a teacher, you could be eligible to receive a scholarship or bursary of up to <%= v :scholarships_generic_maxshortened %> tax-free. This is to support you while you are training and does not need to be paid back.
```